Domain/Entities/DailyDoctorSchedule.cs

```diff
@@ -0,0 +1,14 @@
+namespace Domain.Entities;
+
+public class DailyDoctorSchedule
```

Comment:
Vad ca aici ai si foreign key (DoctorId, LocationId) dar si referinta la tabela Doctor si Location. Acum nu stiu cum e ok sa fie, nu prea am experienta cu Entity Framework sa spun daca e ok sa le ai pe ambele sau ar fi ok doar id-urile.

Reply:
Aici asa trebuie din cate stiu, nu pot configura in OnModelCreating daca nu am si proprietatea de clasa.

Reply:
Okok, il intrebam si pe big boss sa vedem ce zice